src/_utils/handlers/handlers.js

```diff
@@ -138,6 +146,10 @@ export function bracketOfLength(str) {
 export function bracketOfPosition(str) {
     return bracketWithType(str, 'position');
 }
+export function warpToken(str) {
+    if (str.match(/^\$\S/))
+        return `var(--w-${escapeSelector(str.slice(1))})`;
+}
```

Comment:
As this function is very much alike the `cssvar()` besides the `--w` part those two can be refactored and maybe the matching part can be moved to a separate function. But now when I'm writing that I'm not sure if it would improve readability for example. Feel free to disregard my comment but I'll leave it here fwiw